integration/it-integration/src/main/java/se/inera/intyg/rehabstod/integration/it/dto/SickLeavesRequestDTO.java

```diff
@@ -19,12 +19,16 @@
 
 package se.inera.intyg.rehabstod.integration.it.dto;
 
+import java.util.List;
+
 public class SickLeavesRequestDTO {
     private String unitId;
     private String careUnitId;
-    private String doctorId;
+    private List<String> doctorIds;
     private int maxCertificateGap;
     private int maxDaysSinceSickLeaveCompleted;
+    private int toSickLeaveLength;
```

Comment:
We discussed that toSickLeaveLength possibly could be null, should we changed to/from to Integers instead of int so that we can handle null scenarios?